AEFeinstein/Swadge-IDF-5.0
# Lumber multi fix

### Description

Fixed lumberjack multiplayer

### Test Instructions

Ran on two hardware swadges, confirmed they connect, play a game, update each others lives, and finish as expected.

### Ticket Links

n/a

### Readiness Checklist
